graspologic/utils/utils.py

```diff
@@ -360,7 +367,11 @@ def remove_loops(graph: GraphRepresentation) -> Union[np.ndarray, csr_matrix]:
     """
     graph = import_graph(graph)
 
-    dia = diags(graph.diagonal()) if isspmatrix_csr(graph) else np.diag(np.diag(graph))
+    dia = (
```

Comment:
what is this doing? if its already a csr_array you make it a csr_array?

Reply:
scipy still by default returns a csr_matrix but we need a csr_array